### PR TITLE
fix:  removed all hardcoded strings in java files

### PR DESCRIPTION
**Pull request by Bhavnaharitsa**
Fixes: #819
Removed all hardcoded strings from java files.
- [.] Apply the `AndroidStyle.xml` style template to your code in Android Studio.

- [.] Run the unit tests with `./gradlew check` to make sure you didn't break anything

- [.] If you have multiple commits please combine them into one commit by squashing them.

### CODE REVIEW

**Comment by Bhavnaharitsa** [comment]
@shiv07tiwari Please review this

**Comment by shiv07tiwari** [comment]
@Bhavnaharitsa Please don't close the existing PR and make a new one for every change. It causes a lot of unnecessary PRs which is not good for other contributors. Try to make the changes on the same one. You can ask a mentor if you are facing troubles in the same.

File: mifospay/src/main/java/org/mifos/mobilewallet/mifospay/receipt/ui/ReceiptActivity.java [hunk 1]
```diff
@@ -123,12 +123,12 @@ public void showTransactionDetail(Transaction transaction) {
         switch (transaction.getTransactionType()) {
             case DEBIT:
                 isDebit = true;
-                tvOperation.setText("Paid to");
+                tvOperation.setText(R.string.paid_to);
```
Comment (shiv07tiwari) [inline mifospay/src/main/java/org/mifos/mobilewallet/mifospay/receipt/ui/ReceiptActivity.java]: You have not added the getString here.

File: mifospay/src/main/java/org/mifos/mobilewallet/mifospay/receipt/ui/ReceiptActivity.java [hunk 2]
```diff
@@ -123,12 +123,12 @@ public void showTransactionDetail(Transaction transaction) {
...
                 tvOperation.setTextColor(Color.RED);
                 break;
             case CREDIT:
                 isDebit = false;
-                tvOperation.setText("Credited By");
+                tvOperation.setText(R.string.credited_by);
```
Comment (shiv07tiwari) [inline mifospay/src/main/java/org/mifos/mobilewallet/mifospay/receipt/ui/ReceiptActivity.java]: .

File: mifospay/src/main/java/org/mifos/mobilewallet/mifospay/bank/fragment/UpiPinFragment.java [hunk 3]
```diff
@@ -74,7 +74,7 @@ public View onCreateView(@NonNull LayoutInflater inflater, @Nullable ViewGroup c
         if (b != null) {
             step = b.getInt(Constants.STEP, 0);
             upiPin = b.getString(Constants.UPI_PIN, null);
-            mTvTitle.setText("Re-enter UPI PIN");
+            mTvTitle.setText(R.string.reenter_upi);
```
Comment (shiv07tiwari) [inline mifospay/src/main/java/org/mifos/mobilewallet/mifospay/bank/fragment/UpiPinFragment.java]: .

**Comment by Bhavnaharitsa** [comment]
> @Bhavnaharitsa Please don't close the existing PR and make a new one for every change. It causes a lot of unnecessary PRs which is not good for other contributors. Try to make the changes on the same one. You can ask a mentor if you are facing troubles in the same.

No no, I am very much comfortable in squashing commits. It's just that the branch was too clumsy with too many unwanted changes. So had to close it and open a new thing. I'll make sure it doesnt happen again

File: mifospay/src/main/java/org/mifos/mobilewallet/mifospay/receipt/ui/ReceiptActivity.java (hunk 1 above)
Comment (Bhavnaharitsa) [reply mifospay/src/main/java/org/mifos/mobilewallet/mifospay/receipt/ui/ReceiptActivity.java]: Sir, it accepts the ageing right away. Didnt find the need for getString()

**Pull request merged**